jobs/regular/check_akismet_post_voting_comment.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Jobs
+  class CheckAkismetPostVotingComment < ::Jobs::Base
+    def execute(args)
+      return unless SiteSetting.akismet_enabled?
```

Comment:
I think we are trying to move away from unless's. Especially for lines like this
`return unless comment = PostVotingComment.find_by(id: args[:comment_id])` - they are just confusing to read